src/main/java/net/fabricmc/loader/ModInfo.java

```diff
@@ -75,6 +76,9 @@ public String getId() {
 	}
 
 	public String getName() {
+		if(StringUtils.isEmpty(name)){
```

Comment:
Can we drop the unnecessary dependency on Commons here? We want to minimize the amount of dependencies in case Mojang drops them for whatever reason. Just have a "name == null || name.isEmpty()"